README.md

```diff
@@ -22,3 +22,14 @@ You will need:
  grunt platform:add:android
  grunt run
  ```
+
+
+# How to discover resources
+
+Please refer to [here] to learn about detailed steps.
+
+This demo currently supports the following sensors : `Button, Buzzer, Mini Fan, Gas(MQ2), Ambient Light, Led, Motion, RGB Led, Solor, Switch(P) and Temperature`.
+
+After discover the resource, this demo can control the resource and get the expected result.
```

Comment:
More like: "After the discovery process, this demo can read and control the status of the discovered resources."